extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -1319,6 +1313,14 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 					{ value: fileGroup.autogrowAllFiles, enabled: filesCount > 0, style: tableRow, ariaLabel: localizedConstants.AutogrowAllFilesText }
 				]);
 			}
+			else if (fileGroup.type === FileGroupType.FileStreamDataFileGroup && fileGroup.type === filegroupType) {
```

Comment:
nit: put else if on same line as the `}` above for consistency in the repo here and other places in this file